services/director/src/simcore_service_director/config.py

```diff
@@ -17,6 +17,16 @@
 API_VERSION: str = "v0"
 API_ROOT: str = "api"
 
+SERVICE_RUNTIME_SETTINGS: str = "simcore.service.settings"
+SERVICE_REVERSE_PROXY_SETTINGS: str = "simcore.service.reverse-proxy-settings"
+SERVICE_RUNTIME_BOOTSETTINGS: str = "simcore.service.bootsettings"

```

Comment:
minor: I would keep all the constants in a constants module not in the config module.